File: lab6/src/main/java/Bike.java
public class Bike extends Equipment {

    private int gearsNumber;
    private int numberOfBrake;

    public Bike(String ProduceName, double weight, SportType sportType, int gearsNumber, int numberOfBrake) {
        super(ProduceName, weight, sportType);
        this.gearsNumber = gearsNumber;
        this.numberOfBrake = numberOfBrake;
    }

    public int getGearsNumber() {
        return gearsNumber;
    }

    public void setGearsNumber(int gearsNumber) {
        this.gearsNumber = gearsNumber;
    }

    public int getNumberOfBrake() {
        return numberOfBrake;
    }

    public void setNumberOfBrake(int numberOfBrake) {
        this.numberOfBrake = numberOfBrake;
    }

    @Override
    public String toString() {
        return "Bike{" +
                "gearsNumber = " + gearsNumber +
                ", numberOfBrake = " + numberOfBrake + " " +
                super.toString() +
                "}\n";
    }

    @Override
    public String getHeaders() {
        return super.getHeaders()+ "," + "gearsOfNumber" + "," + "numberOfBreak";
    }
    public String toCSV(){
        return super.toCSV() + getGearsNumber() + getNumberOfBrake();
    }
}

File: lab 11(lab 2 python)/bike.py
from equipment import Equipment

class Bike(Equipment):

    def __init__(self, produce_name, weight, producer, gear_numbers, number_of_brake):
        super(Bike, self).__init__(produce_name, weight)
        self.producer = producer
        self.gear_numbers = gear_numbers
        self.number_of_brake = number_of_brake

    def __str__(self):
        return super(Bike, self).__str__() + 'producer = ' + str(self.producer) + 'gear_numbers = ' + str(self.number_of_brake)


